### PR TITLE
405-linkid-potential-bug

### PR DESCRIPTION
**Pull request by DeboraSerra**
## Describe your changes

Briefly describe the changes you made and their purpose.

## Issue number

Mention the issue number(s) this PR addresses (e.g., #123).

## Please ensure all items are checked off before requesting a review:

- [x] I deployed the code locally.
- [x] I have performed a self-review of my code.
- [x] I have included the issue # in the PR.
- [x] I have labelled the PR correctly.
- [x] The issue I am working on is assigned to me.
- [x] I didn't use any hardcoded values (otherwise it will not scale, and will make it difficult to maintain consistency across the application).
- [x] I made sure font sizes, color choices etc are all referenced from the theme.
- [x] My PR is granular and targeted to one specific feature.
- [ ] I took a screenshot or a video and attached to this PR if there is a UI change.

### CODE REVIEW

**Review by erenfn — CHANGES_REQUESTED** [review]
I don't get that error anymore in the db anymore. However, I came across this new error: It occurs when you edit a helper link and delete one of its links:

image
image

File: frontend/src/components/Links/Settings/Settings.jsx
Comment (swoopertr) [inline frontend/src/components/Links/Settings/Settings.jsx:1]: s object imported from "./Settings.module.scss" is may create some inconvenience. It is look like a magical "s".
It should renamed like "ComponentStyle" or "Style". Something like that

**Comment by erenfn** [comment]
I still get this error in the console. It happens when you add a link, delete it, and then clicking save. My guess is that we add that link to the "links to be deleted" list and then try to delete it. However, this error occurs because we haven't created it in the first place. The links that are created before clicking save shouldn't be added to the `deletedLinks` links afterwards

image

**Review by erenfn — APPROVED** [review]

**Review by erenfn — CHANGES_REQUESTED** [review]
The other bug is gone.

However, adding two url's with the same title and deleting one of them deletes both of them. We should fix it

image
image
image

**Review by erenfn — APPROVED** [review]

**Pull request merged**